### PR TITLE
Release notes for WebAuthn

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

**Comment by mposolda** [comment]
@stianst @andymunro Do you have a chance to review this release note please? I think it should go to Keycloak 8 documentation, so I am adding "prioritize" label.

File: release_notes/topics/8_0_0.adoc [hunk 1]
```diff
@@ -30,8 +30,20 @@ SPI was designed for situations when the IDP offer no way to map roles before ad
 
 Keycloak server was upgraded to use WildFly 18 under the covers.
 
+== W3C Web Authentication support
+
```
Comment (andymunro) [inline release_notes/topics/8_0_0.adoc:34]: Should be "a" few limitations.
and "the" contribution.

File: release_notes/topics/8_0_0.adoc [hunk 2]
```diff
@@ -30,8 +30,20 @@ SPI was designed for situations when the IDP offer no way to map roles before ad
...
 = Other Improvements
 
 == System properties and environment variables support in theme.properties
 
 It is now possible to use system properties and environment variables within theme.properties file. Thanks to https://github.com/Opa-[Opa-]
+
+== Support more signing algorithms for client authentication with signed JWT
+
+Thanks to https://github.com/tnorimat[tnorimat], we have support for use more signing algorithms for client authentication with signed JWT.
```
Comment (andymunro) [inline release_notes/topics/8_0_0.adoc]: I would just say "....we support more signing algorithms..."

**Comment by mposolda** [comment]
@andymunro Addressed both your comments. Thanks

**Review by stianst — APPROVED** [review]

**Pull request merged**